platform/linux-generic/m4/configure.m4

```diff
@@ -20,8 +20,6 @@ m4_include([platform/linux-generic/m4/odp_netmap.m4])
 m4_include([platform/linux-generic/m4/odp_dpdk.m4])
 ODP_SCHEDULER
 
-m4_include([platform/linux-generic/m4/performance.m4])
-
```

Comment:
`configure.ac` line 413: `	test_perf_proc:		${test_perf_proc}` still has to be removed.

Reply:
updated in v3.